common/src/services/crypto.service.ts

```diff
@@ -608,14 +619,44 @@ export class CryptoService implements CryptoServiceAbstraction {
 
     // Helpers
 
-    private async shouldStoreKey() {
-        const vaultTimeout = await this.storageService.get<number>(ConstantsService.vaultTimeoutKey);
-        const biometricUnlock = await this.storageService.get<boolean>(ConstantsService.biometricUnlockKey);
+    private async shouldStoreKey(keySuffix: KeySuffixOptions) {
+        let shouldStoreKey = false;
+        if (keySuffix === 'auto') {
```

Comment:
Do we handle a user both having never lock, and biometrics enabled? (Not sure why they would want to)

Reply:
yeah that all works out OK. Actually, most of this PR is handling that exact scenario. Splitting the keys and making sure they're not stored in memory broke never lock way more than biometric :(